docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -68,16 +96,40 @@ SDLMenuCell *cell = [[SDLMenuCell alloc] initWithTitle: <#NSString#> icon: <#SDL
 SDLMenuCell *submenuCell = [[SDLMenuCell alloc] initWithTitle:<#NSString#> icon:<#SDLArtwork?#> subCells:@[cell]];
 self.sdlManager.screenManager.menu = @[submenuCell];
 ```
+###### Core v6.0+
+```objc
+// Create the inner menu cell
+SDLMenuCell *cell = [[SDLMenuCell alloc] initWithTitle:<#NSString#> icon:<#SDLArtwork#> voiceCommands:<#@[NSString]#> handler:^(SDLTriggerSource  _Nonnull triggerSource) {
+    // Menu item was selected, check the `triggerSource` to know if the user used touch or voice to activate it
+    <#Handle the cell's selection#>
+}];
+
+// Create and set the submenu cell
+SDLMenuCell *submenuCell = [[SDLMenuCell alloc] initWithTitle:<#NSString#> icon:<#SDLArtwork?#> submenuLayout:<#Menu Layout#>, subCells:@[cell]];
+self.sdlManager.screenManager.menu = @[submenuCell];
+```
 
 ##### Swift
+###### Pre Core v6.0
```

Comment:
We can just update the code snippets to use the new `init`s as it will still work with head units with pre RPC v6.0. The the tile layout will just not be set.